client/signup/steps/rewind-form/index.jsx

```diff
@@ -0,0 +1,155 @@
+import Card from 'components/card';
+import CompactCard from 'components/card/compact';
+import HappychatButton from 'components/happychat/button';
+import RewindCredentialsForm from 'components/rewind-credentials-form';
+import Popover from 'components/popover';
+import SignupActions from 'lib/signup/actions';
+import { getJetpackCredentials, getJetpackCredentialsUpdateStatus } from 'state/selectors';
+import { recordTracksEvent } from 'state/analytics/actions';
+
+class RewindFormStep extends Component {
+	static propTypes = {
+		flowName: PropTypes.string,
+		goToNextStep: PropTypes.func.isRequired,
+		positionInFlow: PropTypes.number,
+		signupProgress: PropTypes.array,
+		stepName: PropTypes.string,
+	};
+
+	state = {
+		showPopover: false,
+	};
+
+	skipStep = () => this.props.goToNextStep();
+
+	onCancel = () => this.props.goToStep( 'rewind-confirm' );
+
+	goToComplete = () => {
+		SignupActions.submitSignupStep( {
+			processingMessage: this.props.translate( 'Setting up your site' ),
+			stepName: this.props.stepName,
+		} );
+
+		this.props.goToNextStep();
+	};
+
+	togglePopover = () => this.setState( { showPopover: ! this.state.showPopover } );
+	storePopoverLink = ref => ( this.popoverLink = ref );
+
+	renderStepContent = () => {
+		const { happychatEvent, siteId, translate } = this.props;
+
+		return (
+			<div className="rewind-form__container">
+				<h3 className="rewind-form__title">{ translate( 'Site Credentials' ) }</h3>
+				<p>
+					{ translate(
+						"We'll guide you through the process of getting and entering your site's credentials."
+					) }
+				</p>
+				<CompactCard className="rewind-form__section-title">
+					{ translate( 'Enter your credentials' ) }
+				</CompactCard>
+				<Card className="rewind-form__card">
+					<div className="rewind-form__form">
+						<RewindCredentialsForm
+							{ ...{
+								role: 'main',
+								protocol: 'ssh',
+								port: 22,
+								siteId: siteId,
+								allowCancel: true,
+								onCancel: this.onCancel,
+								onComplete: this.props.goToNextStep,
+							} }
+						/>
+					</div>
+				</Card>
+				<CompactCard className="rewind-form__help">
+					<Button
+						className="rewind-form__help-button"
+						borderless={ true }
+						onClick={ this.togglePopover }
+						ref={ this.storePopoverLink }
+					>
+						<Gridicon icon="help" size={ 18 } />
+						{ translate( "Need help finding your site's server credentials?" ) }
+					</Button>
+					<Popover
+						context={ this.popoverLink }
+						isVisible={ this.state.showPopover }
+						onClose={ this.togglePopover }
+						className="rewind-form__help-popover"
+						position="top"
+					>
+						<div>{ translate( 'You can normally get your' ) }</div>
+						<div>{ translate( 'credentials from your hosting' ) }</div>
+						<div>{ translate( 'provider. Their website should' ) }</div>
+						<div>{ translate( 'explain how to get or create the' ) }</div>
+						<div>{ translate( 'credentials you need.' ) }</div>
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'need credentials' )` *ES Score: 11*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/rewind/signup-flow-get-creds)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).